packages/kbn-safer-lodash-set/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@kbn/safer-lodash-set",
-  "version": "0.0.0",
+  "version": "1.0.0",
```

Comment:
Yeah, all packages now have a consistent version as defined by the package linter. If 0.0.0 was useful or intentional I can setup an exception

Reply:
It was set to `0.0.0` because:

1. We don't use versions for internal packages in Kibana as they are just linked
2. I originally wanted to release this to npm as `@elastic/safer-loash-set` (but was voted down 😅), so starting out as `0.0.0` was more correct as it those commits hadn't been released yet (assuming I wanted to release it as `1.0.0`)

However, I don't feel strongly about this, so if you think `1.0.0` is a better "default", feel free to keep it.

Reply:
Yeah, I'm happy to use `0.0.0` across the board, but `@kbn/*` packages won't and can't be published to NPM, they'll have to be converted to `@elastic/*` packages, so then new versioning is reasonable. In this case it was just weird to have version conversations for code which is really only versioned by commit sha. Any commit sha can change anything, as long as the tests pass, so it feels meaningless, just necessary for tooling to not complain.

Reply:
I still want to release it under `@elastic/*` if I can get approval 😅 But in that case the name of the package will have to change anyway, so I'm no longer that concerned with the `version` field.